src/sbvr-api/permissions.ts

```diff
@@ -1064,9 +1062,9 @@ export const checkPermissionsMiddleware = (
 		});
 
 const getGuestPermissions = memoize(
-	() =>
+	async () => {
```

Comment:
Does this need to be `Promise.method` as well for compatibility and consistency with `getReqPermissions`?

Reply:
It does not as this one is only used internally in a `bluebird.join` and meas all returned types remain the same bluebird rather than native promises